widget/playground/src/hooks/useTheme.ts

```diff
@@ -1,39 +1,51 @@
-import { createTheme, theme } from '@rango-dev/ui';
+import {
+  createTheme,
+  darkTheme as defaultDarkTheme,
+  lightTheme as defaultLightTheme,
+} from '@rango-dev/ui';
+import { customizedThemeTokens } from '@rango-dev/widget-embedded';
 import { useEffect, useLayoutEffect, useState } from 'react';
 
 import { NOT_FOUND } from '../constants';
 import { useConfigStore } from '../store/config';
 
 export function useTheme() {
+  const [systemTheme, setSystemTheme] = useState('light');
   const configTheme = useConfigStore.use.config().theme;
   const mode = configTheme?.mode;
 
-  const colors = theme.colors;
+  const { dark, light } = customizedThemeTokens(configTheme?.colors);
 
-  const customLightTheme = createTheme('light-theme-playground', {
-    colors,
-  });
+  const lightThemeClasses = [defaultLightTheme.className];
+  const darkThemeClasses = [defaultDarkTheme.className];
 
-  const customDarkTheme = createTheme('dark-theme-playground', {
-    colors,
-  });
-
-  const [OSTheme, setOSTheme] = useState('light');
+  /*
+   * If theme has been customized, we will push the customized theme to override the default themes.
+   * To be overridden, it should be last thing that has been pushed.
+   */
+  if (light) {
+    const customizedLightTheme = createTheme(light.id, light.tokens);
+    lightThemeClasses.push(customizedLightTheme.className);
+  }
+  if (dark) {
+    const customizedDarkTheme = createTheme(dark.id, dark.tokens);
+    darkThemeClasses.push(customizedDarkTheme.className);
+  }
 
   useEffect(() => {
     const switchThemeListener = (event: MediaQueryListEvent) => {
```

Comment:
Can you verify  that both the widget and playground are loaded with the dark theme when system is in dark mode?

Reply:
yes, i can